cmd/puppet-db/root.go

```diff
@@ -120,10 +125,11 @@ func setCmdFlags(cmd *cobra.Command) {
 	cmd.PersistentFlags().StringP(
 		"config",
 		"c",
-		getDefaultConfig(),
+		"",
 		"`Path` to CLI config, defaults to $HOME/.puppetlabs/client-tools/puppetdb.conf",
```

Comment:
I think test referring to default value should be removed/updated (as it is not valid on windows and not necessarily true)

Reply:
Hmm, it's the same as the rust version. What should we change it to?

Reply:
I propose to set cobra default value to `filepath.Join(usr.HomeDir, ".puppetlabs", "client-tools", "puppet-code.conf")` as this will also update usage text.
Then also adapt config file reading accordingly.